test/Integration.spec.ts

```diff
@@ -5,7 +5,7 @@ import * as childProcess from 'child_process';
 const testUtils = require('../testUtils');
 
 describe('Integration test', function() {
-  this.timeout(30000);
+  this.timeout(50000);
```

Comment:
nextjs dev runner take a bit of time to start up

Reply:
ughh